### DISCUSSION TITLE
libsodium

### DISCUSSION THREAD

**Discussion by bl1nch** [flet-dev/flet#4398] [Packages] [answered]
https://github.com/jedisct1/libsodium
required for `pysodium` (https://github.com/stef/pysodium, https://pypi.org/project/pysodium/)

**Comment by FeodorFitsner** [comment, marked as answer]
pysodium is now supported on both iOS and Android.

**Reply by kentbull** [reply]
Does this mean we can just use the raw libsodium package without having to do a custom loader like I mentioned [here](https://github.com/flet-dev/flet/discussions/4124)?

**Reply by FeodorFitsner** [reply]
Well, lib loading was fixed for iOS and Android, not for macOS.

We need to generally solve that issue for desktop packages, sure. So, you are copying `libsodium.dylib` to some dir inside your app, right?

**Reply by kentbull** [reply]
Yes, and then manually changing the PATH, DYLD_LIBRARY_PATH, and LD_LIBRARY_PATH variables to the directory where the libsodium.dylib file is located so that when the pysodium library loads the dylib that it can see the dylib.
